.github/workflows/unit.yaml

```diff
@@ -1,11 +1,11 @@
-name: Unit tests
+name: CI
 
 on:
   push:
     branches: [ master ]
```

Comment:
I think that we also need to add the tags here... as otherwise this workflow will not be triggered  for tags
```
    tags:
      - v*
```

Reply:
Ok, I think works
```
tags:
      - '*'
```